BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -182,7 +183,10 @@ public final class ConvivaAnalytics: NSObject {
 
         if player.config.sourceItem?.itemTitle == nil && contentMetadataBuilder.assetName == nil {
             throw ConvivaAnalyticsError(
-                "AssetName is missing. Load player source first or set assetName via updateContentMetadata"
+"""
```

Comment:
Is this multi-line string necessary? and not causing any issues at all?

Reply:
Swift lint will complain about the line length. I don't like the multiline string here either. Hm haven't checked the log output format.

Reply:
Output is multiline 👎